src/networking.c

```diff
@@ -2464,7 +2464,10 @@ int processCommandAndResetClient(client *c) {
     int deadclient = 0;
     client *old_client = server.current_client;
     server.current_client = c;
+    long long old_dirty = server.dirty;
     if (processCommand(c) == C_OK) {
+        /* Call command post filters */
+        moduleCallCommandPostFilters(c, server.dirty == old_dirty ? 0 : 1);
```

Comment:
moduleCallCommandFilters is called from within processCommand, so the past trigger should probably be done there as well.
or maybe inside `call` where afterCommand is handled, in which case, care must be taken to make sure it doesn't fire in other cases of `call`, including nested ones)

Reply:
Is this the right place that after [afterCommand] (https://github.com/redis/redis/blob/8d675950e6528e164454300d54aaf7bc024b075c/src/server.c#L3770) within `call` ？

maybe like that: 

> /* Do some maintenance job and cleanup */
>   afterCommand(c);
> /* Call command post filters */
>   moduleCallCommandPostFilters(c, dirty);

Reply:
yes, but it also needs to be conditional.
```c
if (server.execution_nesting == 0)
```
and i suppose:
```c
if (!(c->flags & CLIENT_BLOCKED))
```